tests/sqlparser_bigquery.rs

```diff
@@ -845,10 +845,10 @@ fn parse_like() {
 
         // Test with escape char
         let sql = &format!(
-            "SELECT * FROM customers WHERE name {}LIKE '%a' ESCAPE '\\'",
+            r#"SELECT * FROM customers WHERE name {}LIKE '%a' ESCAPE '\\'"#,
             if negated { "NOT " } else { "" }
         );
-        let select = bigquery().verified_only_select(sql);
+        let select = bigquery().verified_only_select_with_canonical(sql, "");
```

Comment:
as in AST the `ESCAPE` is presented as `escaped_character: char` it can be now only one `char` generically and it won't be never serialised-deserialsed  back to origin `'\\'`. So using `""` as canonical format to compare (skip comparison), BUT keep next `Statement` check to valid it is as expected.

Also this constraint in AST means the valid escpade format of backslash `'\\'` won't be never parsed by dialect with option `ParserOptions::new().with_unescape(false)` as it expect only 1 character by parser rule.